scripts/mina-local-network/mina-local-network.sh

```diff
@@ -181,6 +181,37 @@ spawn-archive-node() {
   exec-archive-node $@ &>${FOLDER}/log.txt &
 }
 
+# Checks connection to database. Give possible reason for lack of connectivity.
+# Function assumes that env have pg_isready command installed. This cli utility is installed along with postgres server
+check-db-connection() {
+  echo "Checking connection to archive database..."
+  printf "\n"
+
+  case `pg_isready -U ${PG_USER} -h ${PG_HOST} -p ${PG_PORT} -d ${PG_DB} &>/dev/null; echo $?` in
+    0)
+      echo "Connection to database successful!"
+      ;;
+    1)
+      echo -n "The server is rejecting connections (for example during startup)"
```

Comment:
Why `-n`? The error message will be followed by the shell prompt.

Reply:
removed